libsolidity/interface/StandardCompiler.cpp

```diff
@@ -1482,15 +1482,18 @@ Json StandardCompiler::compileSolidity(StandardCompiler::InputsAndSettings _inpu
 
 		// IR
 		if (compilationSuccess && isArtifactRequested(_inputsAndSettings.outputSelection, file, name, "ir", wildcardMatchesExperimental))
-			contractData["ir"] = compilerStack.yulIR(contractName);
+			contractData["ir"] = compilerStack.yulIR(contractName).value_or(std::string{});
```

Comment:
Any problems with just using a literal?
```suggestion
			contractData["ir"] = compilerStack.yulIR(contractName).value_or("");
```

Reply:
I wrote it this way because I think it is a bit more idiomatic. One shows the explicit type we're working with and the other relies in implicit conversion. And with `""_s` there's another option. But really I don't think it matters at all. If you prefer `value_or("")` in this case, then let's go with it.

Reply:
I definitely prefer to see the literal when reading the code. I's more concise while still unambiguous. Maybe it's also because a literal stands out more and using an empty string (rather than `nullopt` or something) is often a corner case that people get wrong surprisingly often, so I pay attention to it in review :)

BTW, there are w few more introduced below and in `CommandLineInterface.cpp`.

Reply:
That is fair enough, changed to literal!